libraries/core_libs/consensus/src/pbft/pbft_manager.cpp

```diff
@@ -1068,9 +1072,13 @@ void PbftManager::secondFinish_() {
 
 blk_hash_t PbftManager::generatePbftBlock(const blk_hash_t &prev_blk_hash, const blk_hash_t &anchor_hash,
                                           const blk_hash_t &order_hash) {
-  auto propose_period = pbft_chain_->getPbftChainSize() + 1;
-  const auto pbft_block =
-      std::make_shared<PbftBlock>(prev_blk_hash, anchor_hash, order_hash, propose_period, node_addr_, node_sk_);
+  const auto propose_period = pbft_chain_->getPbftChainSize() + 1;
+  const auto reward_votes = vote_mgr_->getRewardVotes();
+  std::vector<vote_hash_t> reward_votes_hashes;
+  std::transform(reward_votes.begin(), reward_votes.end(), std::back_inserter(reward_votes_hashes),
+                 [](const auto &v) { return v->getHash(); });
+  const auto pbft_block = std::make_shared<PbftBlock>(prev_blk_hash, anchor_hash, order_hash, propose_period,
+                                                      node_addr_, node_sk_, reward_votes_hashes);
```

Comment:
I think we can move `reward_votes_hashes` and make constructor of PbftBlock accept only &&

Reply:
Fixed